### PR TITLE
Share links

### PR DESCRIPTION
**Pull request by gauthierm**
Makes the share buttons on the choice modal window work.

### CODE REVIEW

File: mrburns/main/views.py [hunk 1]
```diff
@@ -42,6 +42,42 @@ def get_context_data(self, **kwargs):
                 'text': _('Join millions of Firefox users around the world '
                           'who are shaping the future of the Web.'),
             }),
+            'share_map_twitter_access': get_tw_share_url({
+                'url': 'https://webwewant.mozilla.org/#web',
```
Comment (pmclanahan) [inline mrburns/main/views.py:46]: Are the url, hashtags, and dnt values always going to be the same? If so we can move those bits to the `get_tw_share_url` function.

File: mrburns/main/views.py [hunk 2]
```diff
@@ -42,6 +42,42 @@ def get_context_data(self, **kwargs):
...
+                'hashtags': '#firefox',
+                'dnt': True,
+                'text': _('ACCESS'),
```
Comment (pmclanahan) [inline mrburns/main/views.py:49]: Do we not have real text for these yet? I'm hesitant to merge this for fear that these strings might be accidentally extracted.

Comment (gauthierm) [inline mrburns/main/views.py:49]: @sgarrity said he'd add real text after this merge. I'll ask him to provide my branch a PR first instead.

**Comment by pmclanahan** [comment]
@gauthierm well if it's ready to go shortly after then that's fine. Just as long as the time w/o the real text is short we should be good.

**Comment by sgarrity** [comment]
> @gauthierm well if it's ready to go shortly after then that's fine. Just as long as the time w/o the real text is short we should be good.

If we've got an r+ on this PR, we can leave it unmerged. I'll get the real strings ready in another branch and merge into master in one elegant, flowing motion.

File: mrburns/main/views.py (hunk 1 above)
Comment (sgarrity) [inline mrburns/main/views.py:46]: I'll have real/final URLs soon and will see how they end up.

**Comment by pmclanahan** [comment]
I think the url code could be consolidated a bit more, but I'm happy to do that after the merge. I didn't go over the non-python bits with a fine-toothed comb, but it looked okay at first glance.

**Comment by gauthierm** [comment]
I'll wait until @sgarrity has the final URLs and clean up the `get_tw_share` function if they're all mostly the same.

**Comment by sgarrity** [comment]
This has been merged manually, though with a slight rebase, so the PR won't show as merged automatically. Nothing to see here. Move along.

**Pull request closed without merging**